### PR TITLE
Added type-ahead filter to Moira list selection dropdown

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/odl-video-service/issues/652

#### What's this PR do?
It adds autocomplete  to select moira list.

#### How should this be manually tested?
Go to http://192.168.99.100:8089/admin/ui/collection/ 
and try to add users in moira list.

@pdpinch 
#### Screenshots (if appropriate)
screen shot 2018-07-16 at 4 44 16 pm

screen shot 2018-07-16 at 4 44 40 pm

### CODE REVIEW

**Comment by amir-qayyum-khan** [comment]
@mbertrand this is django built in auto complete select2 box. I added it after upgrading django. I removed inline. because it do support inlines but it supports one to many or many to many relationships. please take a look.
 
https://docs.djangoproject.com/en/2.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.autocomplete_fields

cc @pdpinch

File: ui/admin.py [hunk 1]
```diff
@@ -24,14 +24,10 @@ class AdminListsInline(admin.TabularInline):
...
     """Customized collection admin model"""
-    inlines = [
-        ViewListsInline,
-        AdminListsInline
-    ]
-    exclude = ('view_lists', 'admin_lists')
     date_hierarchy = 'created_at'
     readonly_fields = ['created_at']
     list_filter = ['stream_source']
+    autocomplete_fields = ['owner', 'view_lists', 'admin_lists']
```
Comment (mbertrand) [inline ui/admin.py]: Also add `autocomplete_fields = ['view_lists']` to `VideoAdmin`

**Comment by amir-qayyum-khan** [comment]
Done @mbertrand

**Comment by amir-qayyum-khan** [comment]
@mbertrand Also i want to merge https://github.com/mitodl/django-elastic-transcoder/pull/4 before this PR

File: requirements.in [hunk 2]
```diff
@@ -41,7 +41,7 @@ django-hijack-admin==2.1.7
 -e git+https://github.com/mitodl/mit-moira.git#egg=mit-moira==0.0.4
 
 # Django-Elastic-Transcoder
--e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
+-e git+https://github.com/mitodl/django-elastic-transcoder.git@8d97d5760eb687ac7fb879a9c1ff1c02e5ae6f6f#egg=django-elastic-transcoder
```
Comment (mbertrand) [inline requirements.in]: Change to:
`-e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder==0.9.7`

File: requirements.txt [hunk 3]
```diff
@@ -6,7 +6,7 @@
 #
 --no-binary psycopg2
 
-git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
+git+https://github.com/mitodl/django-elastic-transcoder.git@8d97d5760eb687ac7fb879a9c1ff1c02e5ae6f6f#egg=django-elastic-transcoder
```
Comment (mbertrand) [inline requirements.txt]: Run `pip-compile` again to update to 0.9.7

**Comment by amir-qayyum-khan** [comment]
Done @mbertrand

**Pull request merged**